src/NameAdress.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Scanner;
+public class NameAdress {
+    public static void main(String[] args) {
+        Scanner perData = new Scanner (System.in);
+        String name, address;
+        System.out.println("What is your name?");
+        name = perData.next();
```

Comment:
why do you use method next()? 
in lecture we have nextLine() and it reads the whole line with spaces
try you code with name and address containing spaces